packages/react-composites/src/composites/ChatComposite/FileDownloadCards.tsx

```diff
@@ -39,7 +41,23 @@ export interface FileDownloadCards {
 export const FileDownloadCards = (props: FileDownloadCards): JSX.Element => {
   const truncateLength = 15;
   const { userId, message } = props;
+  const [showSpinner, setShowSpinner] = useState(false);
   const fileDownloads: FileMetadata[] = message.metadata ? extractFileMetadata(message.metadata) : [];
+  const fileDownloadHandler = useCallback(
+    async (userId, file) => {
+      if (props.downloadHandler) {
+        setShowSpinner(true);
+        const url = await props.downloadHandler(userId, file);
```

Comment:
What will happen if props.downloadHandler throws an error?
I think it should be inside a try catch finally block.

Reply:
done